docs/Access.md

```diff
@@ -0,0 +1,146 @@
+# Access
+
+> Expect these modules to evolve.
+
+Access control—that is, "who is allowed to do this thing"—is incredibly important in the world of smart contracts. The access control of your contract may govern who can mint tokens, vote on proposals, freeze transfers, and many other things. It is therefore critical to understand how you implement it, lest someone else [steals your whole system](https://blog.openzeppelin.com/on-the-parity-wallet-multisig-hack-405a8c12e8f7/).
+
+## Table of Contents
+
+* [Ownable](#ownable)
+  * [Quickstart](#quickstart)
+  * [Ownable library functions](#ownable-library-functions)
+    * [`assert_only_owner`](#assert_only_owner)
+    * [`owner`](#owner)
+    * [`transfer_ownership`](#transfer_ownership)
+    * [`renounce_ownership`](#renounce_ownership)
+  * [Ownable events](#ownable-events)
+    * [`OwnershipTransferred`](#ownershiptransferred)
+
+## Ownable
+
+The most common and basic form of access control is the concept of ownership: there’s an account that is the `owner` of a contract and can do administrative tasks on it. This approach is perfectly reasonable for contracts that have a single administrative user.
+
+OpenZeppelin Contracts provides Ownable for implementing ownership in your contracts.
+
+### Quickstart
+
+Setup simply consists of assigning an owner within the constructor of a Cairo contract like this:
```

Comment:
```suggestion
This library's initializer simply consists of assigning an owner within the constructor of a Cairo contract like this:
```
I'd push extensibility nomenclature and concepts throughout the docs, we cannot expect people to have read other sections so we should at least aid their learning.